src/core/vdom/helpers/normalize-scoped-slots.js

```diff
@@ -64,7 +64,7 @@ function normalizeScopedSlot(normalSlots, key, fn) {
     let vnode: VNode = res && res[0]
```

Comment:
The type annotation suggests that `vnode` is always a `VNode`, but IMO it can also be `undefined`. Otherwise we wouldn't need the `!vnode` check below.